### ISSUE TITLE
Import fine-tuned T5 models (Text-to-SQL) from HF in ONNX format

### ISSUE THREAD

**Issue by AayushSameerShah** [JohnSnowLabs/spark-nlp#14001]
### Is there an existing issue for this?

- [X] I have searched the existing issues and did not find a match.

### Who can help?

I was using `t5_base` model *(around 450MB)* to fine tune on the Spider dataset to make the model able to generate the SQL query from natural language.

### What are you working on?

Working on the SPIDER dataset.

### Current Behavior

After finetuning instead of generating the SQL (even wrong) it generates like:
```
 Es gibt mehrere SQLlite-Tabellen, über deren Eigenschaften in der Folge gesagt wird. # Nirgendwo ist eine Ergebnisanfrage notwendig, für die nur die um die Frage beantworteten Spalten benötigt werden.
```

While the same model *(with pytorch_model.bin and config.json files)* in python with transformers it gives the response for the same prompt:
```
 SELECT sum(SalesData) FROM SalesData WHERE Date DATE = "Christmas"
```

Not a correct one but still it is SQL. **Yes exact the same model**.

### Expected Behavior

It should give the correct response in JSL too.

### Steps To Reproduce

We used the typical pytorch training code to run with 5 epochs and loss around `0.00016220469842664897`.

After that we saved the model: 
```
MODEL_NAME_TF='my_model_tf' # Change metokenizer.save_pretrained(f'./MODEL_NAME_TF}_tokenizer/')
model.save_pretrained(f'./{model}', saved_model=True, save_format='tf')
saved_model_pth = f'./{model}/saved_model/1/assets/'!cp $vocab_pth $saved_model_pth
!shutil.make_archive(f"model", 'zip', f"./{MODEL_NAME_TF}_spark_nlp")
```

So that it could be loaded in JSL

and also saved the model in pytorch so that I could check model loading in python.

### Spark NLP version and Apache Spark

Spark NLP version: 5.1.1
Apache Spark version: 3.2.3

### Type of Spark Application

Python Application

### Additional Information

I should hope it should generate atleast the some query. Because it does in python. 

Thanks.

**Comment by maziyarpanahi** [comment]
We never released any notebook demonstrating how to import T5 or other seq-2-seq models. I currently don't have access to one, but it's not a simple `save_pretrained` like others, hence why it was not released due to complexity of it.

I refer you to. this page which indicates importing T5 in TensorFlow is not supported and the ONNX format is underdevelopment. Once the ONNX is supported it will be an easy 1-line export/import: https://github.com/JohnSnowLabs/spark-nlp/discussions/5669

**Comment by AayushSameerShah** [comment]
Thanks for clarifying mate, but pardon me asking, it's silly to ask but is there any chance you can suggest any other library which I can use in Java to use T5 model?

I mean there are libraries to load the T5 model but the generation is then on us. JSL provides a nice way to control the generation parameters handelling the stuff by itself.

I see DJL doesn't provide anything to run an inference in Java for NLG or I don't find anything which can let me run T5 model in java excel JSL.

Again, it's silly to ask here, but if you can suggest a library?
Thank you so much @maziyarpanahi 🙏🏻

**Comment by maziyarpanahi** [comment]
> Thanks for clarifying mate, but pardon me asking, it's silly to ask but is there any chance you can suggest any other library which I can use in Java to use T5 model?
> 
> I mean there are libraries to load the T5 model but the generation is then on us. JSL provides a nice way to control the generation parameters handelling the stuff by itself.
> 
> I see DJL doesn't provide anything to run an inference in Java for NLG or I don't find anything which can let me run T5 model in java excel JSL.
> 
> Again, it's silly to ask here, but if you can suggest a library? Thank you so much @maziyarpanahi 🙏🏻

You are welcome, we are progressing in making T5 and its family ONNX compatible so users can easily import T5 models (optimized or even quantized). 

As for other libraries, I only know DJL and [tribuo](https://github.com/oracle/tribuo). Not sure they offer seq2seq inference.
